src/main/resources/application.yaml

```diff
@@ -339,6 +339,8 @@ darts:
         max-arm-single-mode-items: 100
       expiry-deletion:
         enabled: ${CASE_EXPIRY_DELETION_ENABLED:false}
+      audio-linking:
+        audio-buffer-seconds: 0s
```

Comment:
probably want to remove the `seconds` part of the parameter name now that its a duration.

Reply:
Updated :)